src/main/java/xbot/common/controls/io_inputs/XCANMotorControllerInputs.java

```diff
@@ -0,0 +1,11 @@
+package xbot.common.controls.io_inputs;
+
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.AngularVelocity;
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class XCANMotorControllerInputs {
```

Comment:
❓ This is maybe a dumb question, but when I read motor controller inputs, I'm thinking things like desired voltage or target speed. Is there another name for these that could distinguish that it's not really the inputs to the motor controller at a meta level but more like inputs to it's outputs?